### ISSUE TITLE
ESPnet2 fine-tuning from Hugging Face

### ISSUE THREAD

**Issue by Miamoto** [espnet/espnet#4453]
Hello all! 

I am fine-tuning a dataset on the below model from ESPnet, which is available on Hugging Face: espnet/simpleoier_librispeech_asr_train_asr_conformer7_hubert_ll60k_large_raw_en_bpe5000_sp. 

1) In the first step I run ./run.sh --skip_data_prep true --skip_train true --download_model espnet/simpleoier_librispeech_asr_train_asr_conformer7_hubert_ll60k_large_raw_en_bpe5000_sp just to evaluate the model in my dataset. I obtain 80.2% of WER on my test set, which is okay for my case scenario. 

2) In the second step I run the run.sh file from my dataset and include --pretrained_model to use the pre-trained valid.acc.ave_10best.pth model. The image below shows my command. The--bpetoken_list and --bpemodel are the variables where I assign the BPE (token file and model respectively) paths of the pre-trained model available on Hugging Face. 

<img width="291" alt="Captura de ecrã 2022-06-15, às 20 01 02" src="https://user-images.githubusercontent.com/15928244/173904485-a8875233-852b-4d36-b5e2-e66513f04ffd.png">

3) After fine-tuning for some epochs the WER increases to 100%, which is wrong. So, I commented on the training loop, to save the uploaded model with the pre-trained weights and perform decoding on top of that. In principle, the model should get the 80.2% since in step 1 that was the WER obtained. But as I perform decoding on the same model, by just initializing the parameters with the command --pretrained_model and using the BPE original token list and model, the performance goes to 100%.

So there is some stranger behaviour when I upload the pre-trained model to the same architecture to perform some fine-tuning on a new dataset. Do you guys know what can be causing this? Shouldn't it be straightforward to perform fine-tuning on a model without this strange behaviour? Thanks a lot!

**Comment by sw005320** [comment]
Can you make sure that the parameters are identical?
If not, please let us know which parameters are different.
Some parts might be unintentionally initialized.

**Comment by Miamoto** [comment]
I already checked, and the parameters are all equal. The only thing I need to remove that the original model does not have is the normalize.mean and normalize.std by setting feats_normalize to empty. This did not influence too much on the WER despite giving a higher WER with feats_normalize activated. The thing that was causing the problem was that I was copying the token_list from the config.yaml file to a txt file and did not notice that I needed to remove all  " and replace \u2581 by this symbol ▁. Thanks for your time!
